src/features/emojiMod.ts

```diff
@@ -65,28 +71,28 @@ const handleReport = (
 };
 
 const reactionHandlers: ReactionHandlers = {
-  "⚠️": (reaction, message, member) => {
+  "⚠️": async (reaction, message, member) => {
     // Skip if the post is from someone from the staff
     if (
       !message.guild ||
       !message.author ||
-      isStaff(message.guild.member(message.author.id))
+      isStaff(message.guild.members.cache.get(message.author.id))
```

Comment:
As I was looking at this, it looked like `.fetch()` was the safer option. Retrieves from cache if available, or loads if not — since this is already async, seems like the one to prefer

```suggestion
      isStaff(await message.guild.members.fetch(message.author.id))
```